src/web/components/ArticleHeadline.stories.tsx

```diff
@@ -59,4 +62,127 @@ export const oldHeadline = () => {
         </Section>
     );
 };
-oldHeadline.story = { name: 'with an old publication date' };
+oldHeadline.story = { name: 'type: basic, with age warning' };
+
+export const Feature = () => {
```

Comment:
This is totally a stylistic thing, but what do you think about losing the curly parens which would allow us to omit the `return`? For example:

```ts
export const Feature = () => (
  <Section>
    ...
```

Reply:
I think yep - this is done

Reply:
For the names, this is a really way of thinking about it - I renamed `opinion` and `feature` to the more descriptive `light` and `bold` which is pretty representative of what they are. It reads a lot better now.

`jumbo` still lives on though, I'm just wary of using words like 'large' or 'big' as they're so generic... Maybe, `large` isn't so bad.... I guess we can see how it sits with the team for now. Anyone reading this in the future who's particularly annoyed by 'jumbo' and wants to change it - feel free 👍